test/jdk/java/awt/dnd/ImageDecoratedDnD/ImageDecoratedDnD.java

```diff
@@ -21,177 +21,79 @@
  * questions.
  */
 
-/*
-  test %W% %E%
-  @bug 4874070
-  @summary Tests basic DnD functionality
-  @author Your Name: Alexey Utkin area=dnd
-  @run applet/manual=yesno ImageDecoratedDnD.html
-*/
-
-import java.applet.Applet;
-import java.awt.*;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Frame;
+import java.awt.Panel;
 import java.awt.dnd.DragSource;
 
+/*
+ * @test
+ * @bug 4874070
+ * @summary Tests basic DnD functionality
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual ImageDecoratedDnD
+ */
 
+public class ImageDecoratedDnD {
+    private static final String INSTRUCTIONS = """
+            A Frame, which contains a yellow button labeled "Drag ME!"
+            and a RED Panel will appear.
+
+            1. Click on the button and drag to the red panel by pressing
+               the "CTRL" key on the keyboard.
+
+            2. When the mouse enters the red panel during the drag, the panel
+               should turn yellow.
+                On the systems that supports pictured drag, the image under the drag-cursor
+                should appear (anchor is shifted, from top-left corner of the picture to
+                inside the picture to 10pt in both dimensions).
+
+                In WIN32 systems the image under cursor would be visible ONLY over
+                the drop targets with activated extended OLE D'n'D support (that are,
+                the desktop and IE).
```

Comment:
@prrace Do we retain the testing instructions related to Win32 as they are quite old?

Reply:
Win32 is not that old… Java still uses these APIs in 64-bit world.

I haven't run the test myself (yet). I suggest replacing _Win32_ with _Windows_. Is image under cursor visible when you drag'n'drop on Windows? It should be.

Reply:
Case 1: A green window below the anchor + circle is seen when CTRL + Mouse is used to perform DnD.
Case 2: A circle + anchor is seen (without a green window) when just Mouse is used to perform DnD.

https://github.com/openjdk/jdk/assets/95945681/4ed2aa47-611c-4957-b930-b86732a95f76

https://github.com/openjdk/jdk/assets/95945681/d8990f71-e90f-4ee2-a7b2-ba5458fbd879

Reply:
You can see the image under the cursor. The image appears when you hover over AWT windows, Firefox window but the image disappears when you hover over Cygwin terminal window.

The image is the translucent blue rectangle with red circle.

Reply:
@aivanov-jdk Updated the instructions to make it more clear. Can you please re-review?